src/main/resources/swagger/api-docs.yaml

```diff
@@ -1966,6 +1966,9 @@ definitions:
         additionalProperties:
           type: string
         description: Map of attributes
+      isProtected:
+        type: boolean
+        description: Whether or not the data should be restricted to dbGaP-enabled users.  Optional, default false.
```

Comment:
This API block in the swagger doc is also missing:
```
required: 
  - namespace
  - name
```
Please add somewhere in here, probably just before line 1957 at the same level as `properties`